conda/gateways/disk/create.py

```diff
@@ -178,6 +178,28 @@ def create_hard_link_or_copy(src, dst):
         shutil.copy2(src, dst)
 
 
+def _do_softlink(src, dst):
+    if is_executable(src):
+        # for extra details, see https://github.com/conda/conda/pull/4625#issuecomment-280696371
+        # We only need to do this copy for executables which have an RPATH containing $ORIGIN
+        #   on Linux, so `is_executable()` is currently overly aggressive.
+        # A future optimization will be to copy code from @mingwandroid's virtualenv patch.
```

Comment:
Oh I did that too already.

https://github.com/conda/conda/pull/4625/commits/b253c467bed429969e2ebd9e7d26a8f0cbe01d25

The full message is there.

I just meant, are you ok with being overly aggressive right now, and then adding in more optimized logic later?